Celeste.Mod.mm/Patches/Monocle/Commands.cs

```diff
@@ -42,6 +43,9 @@ class patch_Commands : Commands {
 
         private static readonly Lazy<bool> celesteTASInstalled = new Lazy<bool>(() => Everest.Modules.Any(module => module.Metadata?.Name == "CelesteTAS"));
 
+        // redirects command logs to the StringBuilder when not null, only set this from main thread
+        public StringBuilder debugRClog;
```

Comment:
Is there a reason to make this available to mods or should it be made `internal` given the restrictions? If there is, this comment should probably be a proper XMLDoc.

Reply:
Probably makes more sense to make this internal, yeah. At least I can't think of a good reason why modders would want to redirect debug command output. The only mod that really does things that technical is CelesteTAS, and that uses its own DebugRC endpoints anyway.